src/docs/sdk/performance/dynamic-sampling-context.mdx

```diff
@@ -0,0 +1,161 @@
+---
+title: "Dynamic Sampling Context (Experimental)"
+---
+
+<Alert level="warning">
+
+This page is under active development.
+Specifications are not final and subject to change.
+Anything that sounds fishy probably is.
+
+</Alert>
+
+Until now, traces sampling was only done through a `traces_sample_rate` option in the SDKs.
+This has quite a few drawbacks for users of Sentry SDKs:
+
+- Changing the sampling rate involved either redeploying applications (which is problematic in case of applications that are not updated automatically, i.e., mobile apps or physically distributed software) or building complex systems to dynamically fetch a sampling rate.
+- Sampling only happened based on a factor of randomness.
```

Comment:
Sampling is happening based on head based, probability sampling (in this case, simple random sampling)

Reply:
Do you have a suggestion on how to reword this? I am kinda lost.

Reply:
Let me think about this and try to push up a commit later today.